.github/actions/tp-e2e-tests/cdn.json

```diff
@@ -0,0 +1,54 @@
+{
+	"hypnotoad": {
+		"listen": [
+			"https://not-a-real-host.test:1?cert=$PWD/localhost.crt&key=$PWD/localhost.key&verify=0x00&ciphers=AES128-GCM-SHA256:HIGH:!RC4:!MD5:!aNULL:!EDH:!ED"
+		],
+		"user": "trafops",
+		"group": "trafops",
+		"heartbeat_timeout": 20,
+		"pid_file": "/var/run/traffic_ops.pid",
+		"workers": 12
+	},
+	"use_ims": true,
+	"traffic_ops_golang": {
+		"insecure": true,
+		"port": "6443",
+		"log_location_error": "error.log",
+		"log_location_warning": "warning.log",
+		"log_location_info": null,
+		"log_location_debug": null,
+		"log_location_event": null,
```

Comment:
Consider logging `log_location_event` so we can see requests made to TO and response status codes